tgui/packages/tgui/interfaces/PreferencesMenu/preferences/gender.ts

```diff
@@ -2,21 +2,27 @@ export enum Gender {
   Male = 'male',
   Female = 'female',
   Other = 'plural',
+  Other2 = 'neuter',
 }
 
 export const GENDERS = {
   [Gender.Male]: {
     icon: 'male',
-    text: 'Male',
+    text: 'He/Him',
   },
 
   [Gender.Female]: {
     icon: 'female',
-    text: 'Female',
+    text: 'She/Her',
   },
 
   [Gender.Other]: {
     icon: 'tg-non-binary',
-    text: 'Other',
+    text: 'They/Them',
+  },
+
+  [Gender.Other2]: {
+    icon: 'tg-non-binary',
```

Comment:
can we not have another icon here to differentiate it?

Reply:
We can, I just don't have one/don't know what we'd use. Any ideas?

Reply:
the neuter symbol from fontawesome would probably work?

Reply:
> the neuter symbol from fontawesome would probably work?

It doesn't match the other symbols, but we can change the others to use the male, female, transgender, and neuter ones. That's an easy change.

Reply:
![image](https://github.com/tgstation/tgstation/assets/7019927/5fa730a3-0542-4fd1-8fd5-411f213d827a)

Looks fine to me.